esp/scm/ws_dfu.ecm

```diff
@@ -707,34 +707,46 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] DFUGetFileMetaDataRe
     int64 TotalResultRows;
 };
 
-
 ESPrequest ListHistoryRequest
 {
-    string name;
+    string Name;
+};
+
+ESPStruct History
+{
+    string Name;
+    string Operation;
+    string Timestamp;
+    string IP;
+    string Path;
+    string Owner;
+    string Workunit;
 };
 
 ESPresponse [exceptions_inline, nil_remove] ListHistoryResponse
 {
-    binary xmlmap;
+    [depr_ver("1.36")] binary xmlmap;
+    ESParray<ESPStruct History> FileHistory;
```

Comment:
@rpastrana @AttilaVamos It wasn't the tag name that bothered me, it was the item name.  History doesn't seem like the name of a single operation.  I also thought the web service should be consistent with the naming of the command line output.